control/setup_control.go

```diff
@@ -52,6 +52,11 @@ type PIDConfig struct {
 	P    float64 `json:"p"`
 	I    float64 `json:"i"`
 	D    float64 `json:"d"`
+
+	// PID block specific values
+	// these are integral sum and signalErr for the pid signal
+	int       float64
+	signalErr float64
```

Comment:
what are these used for in setup?

Reply:
they are not used in setup, they are used within the PID loop. we made them private members so implementers will not try to use them

Reply:
ah my bad I thought this was part of the `PIDLoop` struct so I was confused what their purpose was but this makes more sense